src/app/src/Map.js

```diff
@@ -25,19 +67,28 @@ class GLMap extends Component {
                     scrollZoom={false}
                 >
                     <NavigationControl showZoom={false} />
+                    {Object.values(sensors.features).map(
+                        this.renderCityMarkers
+                    )}
```

Comment:
Could this go all on one line? `{Object.values(sensors.features).map(this.renderCityMarkers)}`

Reply:
Prettier says no.